File: print_array.py
# Question: Given an array of size 5, write code to print the elements - one by one in new line, in any programming language
#way 1:

n=int(input("Enter the length of array:"))
arr=map(int,input("Enter an array consisting of {} elements:".format(n)).split())
for i in arr:
  print(i)
"""
#way 2:
a=[1,2,3,4,5]
for i in a:
  print(i)
"""

File: print_array.cpp
// Given an array of size 5, write code to print the elements - one by one in new line
#include <iostream>
using namespace std;

int main(){
    int len;
    cout<<"input length of array"<<endl;
    //cin>> len;   //input length of array
    len=5;

    int array[1001];
    cout<<"enter the numbers in array"<<endl;
    for(int i=0;i<len;i++){
        cin>>array[i];
    }
    for(int i=0;i<len;i++){
        cout<<array[i]<<endl;
    }
    return 0 ;
}